t/data/fake-shm-module/ngx_http_lua_fake_shm_module.c

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (C) helloyi
```

Comment:
Please remove this since you need to grant your copyright to the authors of this module for this PR to be merged. Thank you.